manifests/install.pp

```diff
@@ -26,8 +26,14 @@
     } else {
       $provider = undef
     }
+
+    $manage_package = $::jmxtrans::package_version ? {
```

Comment:
After setting the default value for `$::jmxtrans::package_version` to `'present'`, this is unnecessary.